schema/1_add_tsvector_to_clubs.sql

```diff
@@ -8,6 +8,7 @@ CREATE TABLE clubs (
     instagram_url TEXT,
     twitter_url TEXT,
     created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT (now() at time zone 'utc')
+    search_vector TSVECTOR
```

Comment:
Is `TSVECTOR` an actual Postgresql datatype? I've searched around and it doesn't seem like a real datatype, but rather a useful query procedure.